dotnet-estimation/dotnet-estimation/Templates/WebAPI/Devon4Net.Application.WebAPI.Implementation/Business/SessionManagement/Exceptions/NoOpenOrSuppendedTask.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Devon4Net.Application.WebAPI.Implementation.Business.SessionManagement.Exceptions
+{
+    public class NoOpenOrSuppendedTask : Exception
```

Comment:
There's a typo in the class name `NoOpenOrSuspendedTask`

Reply:
changed the code as suggested